stache.cabal

```diff
@@ -1,5 +1,5 @@
 name:                 stache
-version:              1.1.2
+version:              1.1.3
```

Comment:
The same here, should be 1.2.0, although it'd be better not to bump in PR, I'll bump myself before release.